gradle/changelog/file_names_with_hashtags.yaml

```diff
@@ -0,0 +1,4 @@
+- type: fixed
+  description: Fix file edit for file names with hashtags ([#65](https://github.com/scm-manager/scm-editor-plugin/pull/65))
```

Comment:
It's not hashtags, it's hashes.

Reply:
Thank you, sometimes my English skills are lacking.